packages/ui/src/1_atoms/Tabs/Tabs.types.ts

```diff
@@ -0,0 +1,15 @@
+export enum TabSize {
+  small = 'small',
+  normal = 'normal',
+}
+
+export enum TabStyle {
+  primary = 'primary',
+  secondary = 'secondary',
+}
+
+export enum TabColor {
```

Comment:
I don't like the fact it contains `buy` and `sell` as it implies the usage (which is wrong in this case anyway). Btw, we discussed we would get rid of this enum, didn't we?